examples/article-access.amp.html

```diff
@@ -179,11 +179,11 @@ <h1 itemprop="headline">Lorem Ipsum</h1>
           </div>
         </div>
 
-        <section class="login-section">
+        <section amp-access="NOT access" class="login-section">
           <a on="tap:amp-access.login">Login to read more!</a>
         </section>
 
-        <div class="article-body" itemprop="articleBody">
+        <div amp-access="access" class="article-body" itemprop="articleBody">
```

Comment:
I like `allow`/`deny`, however I need to think about it. Expressions are fairly open and do not necessarily have semantics of "allow/deny". E.g. "access" comes from the user defined response and could mean anything - the section is visible of hidden depending on whether this expression evaluates to `true`.